src/OpenSage.Game/Graphics/Cameras/Camera.cs

```diff
@@ -86,7 +86,7 @@ public Camera(Func<Viewport> getViewport)
             BoundingFrustum = new BoundingFrustum(Matrix4x4.Identity);
 
             _nearPlaneDistance = 4.0f;
-            _farPlaneDistance = 2000.0f;
+            _farPlaneDistance = 10000.0f;
```

Comment:
This reverts the chang from the water PR, but the short plane distance eventually got annoying and this also broke the shadow mapping